server/app/services/mongodb/migrator.rb

```diff
@@ -68,13 +69,16 @@ def migrations
     def migrate
       ensure_indexes
       release_stale_lock
-      with_dlock(LOCK_NAME, LOCK_TIMEOUT) do
-        migrate_without_lock
-      end
+      lock_id = wait_until!("lock #{name} is available", timeout: LOCK_TIMEOUT, interval: 0.5) { 
```

Comment:
`/home/kontena/kontena/kontena/server/app/services/mongodb/migrator.rb:72:in 'migrate': undefined local variable or method 'name' for #<Mongodb::Migrator:0x00000002a933a8> (NameError)`